targets/targets.json

```diff
@@ -4168,6 +4208,10 @@
             "lse_drive_load_level": {
                 "help": "HSE drive load level  RCC_LSEDRIVE_LOW | RCC_LSEDRIVE_MEDIUMLOW | RCC_LSEDRIVE_MEDIUMHIGH | RCC_LSEDRIVE_HIGH",
                 "value": "RCC_LSEDRIVE_LOW"
+            },
+            "i2c_timing_value_algo": {
+                "help": "If value was set I2C timing algorithm is enabled. Enabling may leads to performance issue. Keeping this false and changing system clock will cause compilation error)",
+                "value": false
```

Comment:
If this is false (default value), and I change system clock, why do I get compilation error?

Reply:
In this scenario, there will no error while compiling. But you may get an ASSERT while running.

Reply:
@affrinpinhero-2356  please update help description accordingly

Reply:
modified help description.

Reply:
ok thanks